forest-node/terraform/Makefile

```diff
@@ -0,0 +1,11 @@
+init:
+	@terraform init" 
```

Comment:
Is this a typo? I can't seem to make it work.

Reply:
After removing the `"`, I get this error:
```
Initializing the backend...
╷
│ Error: error configuring S3 Backend: no valid credential sources for S3 Backend found.
│ 
│ Please see https://www.terraform.io/docs/language/settings/backends/s3.html
│ for more information about providing credentials.
│ 
│ Error: NoCredentialProviders: no valid providers in chain. Deprecated.
│ 	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
│ 
│ 
│ 
```

I've already configured `s3cmd`:
```
$ s3cmd ls
2022-07-12 13:29  s3://forest-snapshots
2021-11-26 19:23  s3://lodestar-misc
```

Reply:
I've made some changes in regards to `terraform init` but have you already created the space bucket and attached the name to the `backend.tf` file ?